dev/io.openliberty.jakartaee9.internal_fat/fat/src/io/openliberty/jakartaee9/internal/tests/JakartaEE9Test.java

```diff
@@ -0,0 +1,95 @@
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.RepeatTestFilter;
+import componenttest.rules.repeater.FeatureReplacementAction;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import componenttest.topology.utils.PrivHelper;
+import io.openliberty.jakartaee9.internal.apps.jakartaee9.web.WebProfile9TestServlet;
+
+@RunWith(FATRunner.class)
+public class JakartaEE9Test extends FATServletClient {
+
+    @ClassRule
+    public static RepeatTests repeat = RepeatTests
+                    .with(new FeatureReplacementAction()
+                                    .removeFeature("jakartaee-9.1")
+                                    .removeFeature("webProfile-9.0")
+                                    .removeFeature("webProfile-9.1")
+                                    .addFeature("jakartaee-9.0")
+                                    .withID("jakartaee90")) //LITE
+                    .andWith(new FeatureReplacementAction()
+                                    .removeFeature("jakartaee-9.0")
+                                    .removeFeature("jakartaee-9.1")
+                                    .removeFeature("webProfile-9.1")
+                                    .addFeature("webProfile-9.0")
+                                    .withID("webProfile90")
+                                    .fullFATOnly())
+                    .andWith(new FeatureReplacementAction()
+                                    .removeFeature("jakartaee-9.0")
+                                    .removeFeature("webProfile-9.0")
+                                    .removeFeature("webProfile-9.1")
+                                    .addFeature("jakartaee-9.1")
+                                    .withID("jakartaee91")
+                                    .fullFATOnly())
+                    .andWith(new FeatureReplacementAction()
+                                    .removeFeature("jakartaee-9.0")
+                                    .removeFeature("jakartaee-9.1")
+                                    .removeFeature("webProfile-9.0")
+                                    .addFeature("webProfile-9.1")
+                                    .withID("webProfile91")
+                                    .fullFATOnly());
+
+    public static final String APP_NAME = "webProfile9App";
+
+    @Server("jakartaee9.fat")
+    @TestServlet(servlet = WebProfile9TestServlet.class, contextRoot = APP_NAME)
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        PrivHelper.generateCustomPolicy(server, PrivHelper.JAXB_PERMISSION);
```

Comment:
This should not be needed since we don't use com.sun.xml.bind any longer for XMLBinding 3.0

Reply:
removed